docker-compose.yml

```diff
@@ -0,0 +1,72 @@
+app:
+    image: debian:jessie
+    volumes:
+        - ./:/var/www/symfony
+    tty: true
+
+frontend:
+    build: docker/frontend
+    volumes_from:
+        - app
+
+db:
+    image: mysql
+    ports:
+        - 3306
+    environment:
+        MYSQL_ALLOW_EMPTY_PASSWORD: 'yes'
+        MYSQL_ROOT_PASSWORD: ''
+        MYSQL_DATABASE: symfony
+        MYSQL_USER: symfony
+        MYSQL_PASSWORD: symfony
```

Comment:
you should link a volumes with a simple name:

```
volumes:
    - mysqldata:/var/lib/mysql
```

This way you can delete all your containers without losing data of mysql and still be environment agnostic